### PR TITLE
Reworking Getting Started

### PR DESCRIPTION
**Pull request by HelgaMond**
PR for the issue #26. 
What was done:
- revised getting started structure;
- getting started introduction was written;
- jtango-maven-archetype was added;
Have added stub sections. 
Hope when this PR is merged other members will contribute to corresponding sections (e.g. Python developers to Python development).

### CODE REVIEW

**Comment by HelgaMond** [comment]
@piogor could you please review and merge this PR. I have done with it. In addition to the mention points I have moved some information from the tango-controls.org

**Comment by piogor** [comment]
@HelgaMond ,Hi Olga, it seems that "How to install Tango on various platform went out of Getting Started menu level :
obraz
Is it on purpose or accidental?
Then, have you seen pending review comments? 
If you wish, I may do appropriate corrections by my self before merging.

File: source/getting-started/end-user/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.. _getting_started_as_user:
+
+End-user applications guide
+===========================
+
+Jive
```
[the remark this answers is not in the document]
Comment (HelgaMond) [reply source/getting-started/end-user/index.rst]: @piogor 
Piotr, thank you for the quick comments! 
1. Yes, I have decided to make a separate menu item "How to install Tango ..." as it was difficult to find this information. It seems logical: first, you read overview; second, you install and then read what to do with all that stuff in "Getting started".
2. Yes, please, if it is not difficult for you, make these corrections before merging. I completely agree with you. Thank you!
3. Concerning " Tango Controls or Tango-Controls". Another question is how to write "Tango" itself?  The whole word in capitals or only the first symbol? And is it  only "Tango" or "Tango controls"? "Tango controls" makes quicker associations with the GitHub and the web site. @andygotz Andy, what you you think?

[the remark this answers is not in the document]
Comment (piogor) [reply source/getting-started/end-user/index.rst]: So, I will do corrections.

[the remark this answers is not in the document]
Comment (piogor) [reply source/getting-started/end-user/index.rst]: Just for reference: it makes important the following tasks: #21 (Add LogViewer documenation) and #42 (ATKPanel documentation)

File: source/getting-started/administration/index.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.. _getting_started_as_administrator:
+
+Administration applications guide
+=================================
+
+Astor
+~~~~~
+
```
Comment (piogor) [inline source/getting-started/administration/index.rst]: Here I will add links, too (as for end-user index).

**Comment by HelgaMond** [comment]
@piogor Piotr, thank you very much for your comments and input!

**Pull request merged**